eisbuk-admin/src/components/atoms/AttendanceSheet/AttendanceSheetSlot.tsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+
+import TableBody from "@material-ui/core/TableBody";
+import TableRow from "@material-ui/core/TableRow";
+import Table from "@material-ui/core/Table";
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+
+import makeStyles from "@material-ui/core/styles/makeStyles";
+
+import { SlotInterface } from "eisbuk-shared";
+import { CustomerWithAttendance } from "@/types/components";
+
+import { getSlotTimespan } from "@/utils/helpers";
+
+/** @TODO remove this and import same type when it's merged*/
+export interface Props extends SlotInterface {
+  /**
+   * Record of customers who have booked (or are manually as attended),
+   * keyed by `customerId` and having all of the data for the customer plus
+   * values for `bookedInterval` and `attendedInterval`.
+   */
+  customers: CustomerWithAttendance[];
+}
+
+const AttendanceSheetSlot: React.FC<Props> = ({
+  customers,
+  intervals,
+  type,
+  notes,
+}) => {
+  const classes = useStyles();
+
+  const timeString = getSlotTimespan(intervals);
+
+  /** UI todos:
+   * @TODO in page component render alternative background shades for slot table
+   * @TODO center each slot timestring (different width percentages for table cells doesn't work)
+   */
+  return (
+    <Table aria-label="simple table">
+      <TableHead>
+        <TableRow className={classes.slotHeading}>
+          <TableCell className={classes.bold}>
+            <span>{timeString} </span>
+            <span> {type.toUpperCase()}</span>
+            {notes && <span> ({notes})</span>}
```

Comment:
Ok, so between these spans, I would create some margin. Also, maybe it would be good to use type badge (like in the slot component).